web/ui/src/features/clustering/PeerList.tsx

```diff
@@ -0,0 +1,45 @@
+import { PeerInfo } from '../clustering/types';
+
+import Table from './Table';
+
+import styles from './PeerList.module.css';
+
+interface PeerListProps {
+  peers: PeerInfo[];
+}
+
+const TABLEHEADERS = ['Node Name', 'Advertised Address', 'Current State', 'Local Node'];
+
+const PeerList = ({ peers }: PeerListProps) => {
+  const tableStyles = { width: '130px' };
+
+  /**
+   * Custom renderer for table data
+   */
+  const renderTableData = () => {
+    return peers.map(({ name, addr, state, isSelf }) => (
+      <tr key={name} style={{ lineHeight: '2.5' }}>
+        <td>
+          <span className={styles.idName}>{name}</span>
+        </td>
+        <td>
+          <span className={styles.idName}>{addr}</span>
+        </td>
+        <td>
+          <span className={styles.idName}>{state}</span>
+        </td>
+        <td>
+          <span> {isSelf ? '✅' : '🚫'}</span>
```

Comment:
Is there anyway to get the distribution of activity? Might be entirely out of scope but I think dskit has something like it.

Reply:
Hm, no, since there's no central KV store but each node uses the hashing function whenever it needs to, there's currently no way to estimate what the current distribution is.

A potentially interesting thing here is to be able to list the components that make use of clustering, calculate the percentage of targets owned and navigate to their details page for more fine-grained information.

Reply:
That's a fair trade off. Might be something to consider in the future having each node expose debug info that's not shared via ckit.